challenge-07/challenge-07.js

```diff
@@ -1,51 +1,94 @@
 /*
 Crie um array com 5 items (tipos variados).
 */
-// ?
+
+var arr = [ 1, 'Max', true, null, { carro: 'verde' }]
 
 /*
 Crie uma função chamada `addItem`, que irá adicionar itens no array criado.
 A função deverá retornar o array atualizado.
 */
-// ?
+
+function addItem (item) {
+  arr.push (item);
+}
 
 /*
 Adicione um novo array ao array criado no início do desafio, com ao menos 3
 itens de tipos diferentes, mostrando o resultado no console.
 */
-// ?
+
+addItem([ false, function() {}, 20 ]);
+
+console.log(arr);
 
 /*
 Mostre no console o segundo elemento desse último array, criado acima, com a
 frase:
 "O segundo elemento do segundo array é [ELEMENTO]."
 */
-// ?
+
+console.log(
+
+  'O segundo elemento do segundo array é ' + arr[5][1] + '.'
+
+  );
 
 /*
 Mostre no console quantos itens tem o primeiro array criado, com a frase:
 "O primeiro array tem [QUANTIDADE DE ITENS] itens."
 */
-// ?
+console.log(
+
+  'O primeiro array tem ' + arr.length + ' itens.'
+
+  );
 
 /*
 Agora mostre no console quantos itens tem o segundo array criado, com a frase:
 "O segundo array tem [QUANTIDADE DE ITENS] itens."
 */
-// ?
+
+console.log(
+
+  'O segundo array tem ' + arr[5].length + ' itens.'
+
+  );
 
 /*
 Utilizando a estrutura de repetição `while`, mostre no console todos os números
 pares entre 10 e 20, inclusive esses 2.
 */
+// var numeros = 10;
+
+// while(numeros <= 20 || numeros % 2 = 0) {
+//   console.log( numeros++ );
+// }
+
+// porque assim não funciona?
+// Já entendi como fazer, mas ainda não entendi o que está errado no código acima.
```

Comment:
@maxarouca, você está dizendo o seguinte no seu `while`:

**ENQUANTO** `numeros` for menor ou igual a `20` **OU** `numeros` dividido por 2 **receber** (olha o sinal de igual ali) zero, **FAÇA ALGO...**

Achou o erro? xD